drizztsaga/tpa/ds-nerfing.tpa

```diff
@@ -0,0 +1,228 @@
+// ---------------------------------------------------------------------------------------------------
+DEFINE_ACTION_MACRO ~DS_ITEMS_NERFING~ BEGIN
+// ---------------------------------------------------------------------------------------------------
+
+// +5 weapons into +3
+ACTION_FOR_EACH ~file~ IN
+				~F_ADAGGR~ // Snake Soul +5
+				~F_ICEAX5~ // Ice Slasher +5
+				~F_SW2H17~ // Belhifet's Sword of Grief +5
+BEGIN
+  COPY_EXISTING ~%file%.itm~ ~override~ // nerfing +5 weapon into "only" +3 weapon
+	WRITE_SHORT 0x60 3 // Enchantment Level
+    LPF ALTER_ITEM_HEADER INT_VAR type = 1 thac0_bonus = 3 damage_bonus = 3 END
+	PATCH_FOR_EACH offset IN 0xc 0x54 BEGIN
+	  READ_LONG ~%offset%~ desc_strref
+	  PATCH_IF (desc_strref >= 0 && desc_strref < 2147483646) BEGIN
+		READ_STRREF ~%offset%~ desc
+		INNER_PATCH_SAVE desc ~%desc%~ BEGIN
+		  REPLACE_TEXTUALLY CASE_INSENSITIVE EXACT_MATCH ~+ 5~ ~+ 3~ // change thac0 and damage modifier in description to + 3
```

Comment:
The culpit is F_ICEAX5.itm // Ice Slasher +5

The main.tra could be updated but it is how it is format in oBG2 for weapons : 1dx + x

In fact, now you say that, I think, some weapons formating for obg2 are not good. I will look on that in tra files

Reply:
> Well, that's a shame. A good approach would be to check if BG2-fixpack fixed such spacing in the GTU components. For some languages, the format `+ x` is generally wrong. I don't sure how is it work with English. In this case, I would update DS .tra file to be constant with correct format instead of reproducing the same errors, even if they are still in oBG2.

It is not wrong but as it is in oBG2. I have update tra file for oBG2 according to that.

Reply:
> A good approach would be to check if BG2-fixpack fixed such spacing in the GTU components. For some languages

As it is how it orks on oBG2, it is not `fixed` in gtu. changing it in tra file of DS (for BG not BGEE) would make all description non coherent with oBG2.

FWIW, for EE, it is already as you said

Reply:
That's fine, I think. Thanks! It is important to run the scripts for all combination.